### ISSUE TITLE
mxnet fine tuned model, result wrong

### ISSUE THREAD

**Issue by Shiro-mx** [dmlc/mxnet#5532]
Hello,

I would like to fine tuned a pre-trained model VGG16 ( pretrained on different dataset). I try to fine tuned it fixing all the layers excepted the full connected layers. But I obtain wrong result.  In fact I obtain the same validation result for each epoch.

```
 ## Data train
    train_dataiter = mx.io.ImageRecordIter(
        path_imgrec=imgrec_train,
        data_shape=(3,224,224),
        batch_size=1,
        path_imglist=imglist_train,
        preprocess_threads=1,
        label_width=label_width,
        data_name='data',
        label_name='softmax_label'
    )

    # Data validation
    val_dataiter = mx.io.ImageRecordIter(
        path_imgrec=imgrec_val,
        data_shape=(3,224,224),
        batch_size=1,
        path_imglist=imglist_val,
        preprocess_threads=1,
        label_width=label_width,
        data_name='data',
        label_name='softmax_label'
    )

prefix = 'mxnet_test_learning'
frequent = 500
batch_end_callback = mx.callback.Speedometer(train_dataiter.batch_size, frequent=frequent)
 epoch_end_callback = mx.callback.do_checkpoint(prefix)

# Load the model
model = mx.mod.Module.load(name_model, epoch, context=mx.gpu(0), data_names=['data'] ,label_names=['softmax_label'], fixed_param_names=['conv1_1','relu1_1','conv1_2','relu1_2','pool1','conv2_1','relu2_1','conv2_2','relu2_2','pool2','conv3_1','relu3_1','conv3_2','conv3_3','relu3_3','pool3','conv4_1','relu4_1','conv4_2','relu4_2','conv4_3','relu4_3','pool4','conv5_1','relu5_1','conv5_2','relu5_2','conv5_3','relu5_3','pool5','flatten_0'])

learning_rate_schedul =  mx.lr_scheduler.FactorScheduler(1,0.9)
model.fit(train_data=train_dataiter, eval_data=val_dataiter,eval_metric='mae' ,  
            optimizer='sgd', optimizer_params={'learning_rate':0.001, 'momentum': 0.9, 'wd':0.0005},
            force_init=False, begin_epoch=0, num_epoch=10, epoch_end_callback=epoch_end_callback, batch_end_callback = batch_end_callback, kvstore='local' )

```

Have I forgot some parameters? Where am I wrong? I change the value of wd and learning rate but without any success.

**Comment by sunbinbin1991** [comment]
Have you solve your problem, I have  question similar to you, as for me ,my "mae" always have no change, but when I use "acc" instead, each epoch could make change but just little
